specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json

```diff
@@ -2048,6 +2071,31 @@
               "type": "string",
               "description": "Impacted resource region name."
             },
+            "resourceName": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Resource name of the impacted resource."
+            },
+            "resourceGroup": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Resource group name of the impacted resource."
+            },
+            "maintenanceStatus": {
```

Comment:
Should it be status or maintenanceStatus?

Reply:
Srinath confirmed that it should be status.